### PR TITLE
Updates for ITKv5.2 code cleanups ... ROUND1

### PR DESCRIPTION
**Pull request by hjmjohnson**

### CODE REVIEW

File: src/Developer/ImageFilterMultipleOutputs.h [hunk 1]
```diff
@@ -35,7 +35,7 @@ class ImageFilterMultipleOutputs : public ImageToImageFilter<TImage, TImage>
 
   /**  Create the Output */
   DataObject::Pointer
-  MakeOutput(ProcessObject::DataObjectPointerArraySizeType idx);
+  MakeOutput(ProcessObject::DataObjectPointerArraySizeType idx) override;
```
Comment (dzenanz) [inline src/Developer/ImageFilterMultipleOutputs.h]: Misleading commit message

**Comment by dzenanz** [comment]
Looks good

**Comment by dzenanz** [comment]
The tests still fail :(

**Comment by hjmjohnson** [comment]
> The tests still fail :(

Did the test pass previously?  Where are the for the pipeline?  I'm guessing there is a problem with the environment setup.

**Comment by dzenanz** [comment]
This looks like a missed enum update:
```
/home/vsts/work/1/s/src/Core/Common/Transparency/Code.cxx:86:29: error: 'itk::IOPixelEnum' has not been declared
   tiffIO->SetPixelType(itk::IOPixelEnum::RGBA);
```

**Comment by dzenanz** [comment]
Previously, maybe 80 tests failed. Now 300 fail.

**Comment by dzenanz** [comment]
Wait for CI result before deciding whether to merge.

**Comment by hjmjohnson** [comment]
> The builds seem to fail due to not getting the test data. Example message:
> 
> ```
> CMake Error at /home/vsts/work/1/s/CMake/ExternalData.cmake:1113 (message):
>   
> 
>   Object
>   SHA512=853998d046f52aded5b7e3293ab356a156339a85488e49735435fa9b2574ae248859fcd3988e1fe07a728441edabdc2bf207883c45b3f5862ded54d931f46caf
>   not found at:
> 
>     https://insightsoftwareconsortium.github.io/ITKTestingData/SHA512/853998d046f52aded5b7e3293ab356a156339a85488e49735435fa9b2574ae248859fcd3988e1fe07a728441edabdc2bf207883c45b3f5862ded54d931f46caf ("HTTP response code said error")
>     https://data.kitware.com:443/api/v1/file/hashsum/SHA512/853998d046f52aded5b7e3293ab356a156339a85488e49735435fa9b2574ae248859fcd3988e1fe07a728441edabdc2bf207883c45b3f5862ded54d931f46caf/download ("Server returned nothing (no headers, no data)")
> ```

I found a cache of local objects on my system, and found the requested hash.  I uploaded thee requested hash.

**Pull request merged**